website/Contribution/CONTRIBUTING.rst

```diff
@@ -1095,16 +1095,14 @@ http://en.wikipedia.org/wiki/Software_peer_review.
 Please respect a few basic rules:
 ---------------------------------
 
-* Two reviewers must approve a merge proposal in order to be able to merge it
-* 5 calendar days must be given to be able to merge it
-* A PR can be merged in less than 5 calendar days if and only if it is approved
-  by 3 reviewers. If you are in a hurry just send a mail at
-  contributors@odoo-community.org or ask by IRC (FreeNode
-  oca, openobject channel).
+* Read `OCA Maturity Levels and Development Status Policy <https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst#oca-maturity-levels-and-development-status-policy-20180824>`_.
```

Comment:
s/Read/Read and follow the rules stated for the module maturity level

Reply:
@pedrobaeza done